check-before-deployment.md

```diff
@@ -0,0 +1,350 @@
+    nvme0n1
+    └─nvme0n1p1 ext4         c51eb23b-195c-4061-92a9-3fad812cc12f
+    ```
+
+5. Edit the `/etc/fstab` file and add the `nodelalloc` mount options.
+
+    {{< copyable "shell-root" >}}
+
+    ```bash
+    vi /etc/fstab
+    ```
+
+    ```
+    UUID=c51eb23b-195c-4061-92a9-3fad812cc12f /data1 ext4 defaults,nodelalloc,noatime 0 2
+    ```
+
+6. Mount the data disk.
+
+    {{< copyable "shell-root" >}}
+
+    ```bash
+    mkdir /data1 && \
+    mount -a
+    ```
+
+7. Check using the following command.
+
+    {{< copyable "shell-root" >}}
+
+    ```bash
+    mount -t ext4
+    ```
+
+    ```
+    /dev/nvme0n1p1 on /data1 type ext4 (rw,noatime,nodelalloc,data=ordered)
+    ```
+
+    If the filesystem is ext4 and `nodelalloc` is included in the mount options, you have successfully mount the data disk ext4 filesystem with options on the target machines.
+
+## Check and disable system swap
+
+This section describes how to disable swap.
+
+TiDB requires sufficient memory space for operation. It is not recommended to use swap as a buffer for insufficient memory, which might reduce performance. Therefore, it is recommended to disable the system swap permanently.
+
+Do not disable the system swap by executing `swapoff -a`, or this setting will be invalid after the machine is restarted.
+
+To disable the system swap, execute the following command:
+
+{{< copyable "shell-regular" >}}
+
+```bash
+echo "vm.swappiness = 0">> /etc/sysctl.conf 
+swapoff -a && swapon -a
+sysctl -p
+```
+
+## Check and stop the firewall service of deployment machines
+
+This section describes how to stop the firewall service of deployment machines.
+
+In TiDB clusters, the access ports between nodes must be open to ensure the normal transmission of information such as read and write requests and data heartbeat. In common online scenarios, the data interaction between the database and the application service and between the database nodes are all made within a secure network. Therefore, if there are no special security requirements, it is recommended to stop the firewall of the target node. Otherwise, refer to [the port usage](/hardware-and-software-requirements.md#network-requirements) and add the related port information to the whitelist of the firewall service.
+
+1. Check the firewall status. Take CentOS Linux release 7.7.1908 (Core) as an example.
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    sudo firewall-cmd --state
+    sudo systemctl status firewalld.service
+    ```
+
+2. Stop the firewall service.
+
+    {{< copyable "shell-regular" >}}
+
+    ```bash
+    sudo systemctl stop firewalld.service
+    ```
+
+3. Disable automatic start of the firewall service.
+
+    {{< copyable "shell-regular" >}}
+
+    ```bash
+    sudo systemctl disable firewalld.service
+    ```
+
+4. Check the firewall status.
+
+    {{< copyable "shell-regular" >}}
+
+    ```bash
+    sudo systemctl status firewalld.service
+    ```
+
+## Check and install the NTP service
```

Comment:
Please align this [commit](https://github.com/pingcap/docs-cn/pull/3457/commits/725de1d56ae3e0081c3b51d8b68cd28380912700). @ran-huang

Reply:
Updated.